### ISSUE TITLE
Support for vscode extension project written by typescript

### ISSUE THREAD

**Issue by syhxzzz** [NiGhTTraX/ts-monorepo#347]
Hi,

I've recently been developing a VS Code extension using a monorepo setup, and your solution has been very inspiring. I used your approach to make my project work seamlessly with VS Code extension development. You can find a sample demo in my repo: [vscode-extension-monorepo](https://github.com/syhxzzz/vscode-extension-monorepo/tree/main).

My main changes were to modify the tsconfig.json and webpack.config.ts files to ensure everything works correctly. I was wondering if it would be okay to add a VS Code extension example to your /app directory? If you're open to it, I can redesign my code accordingly and submit a pull request.

Thank you for your time and consideration.

**Comment by NiGhTTraX** [comment]
Hey @syhxzzz, glad to hear you've found the project useful!

I like to keep the examples included in this project to a minimum and focus on toolchains (bundlers, transpilers, test runners etc.), rather than on specific targets (serverless, extensions etc.). This makes maintenance for me easier and gives people a foundation they can build upon.

Could you summarize the changes you've made, particularly around configs needed for monorepo/path aliases? I skimmed through the repo you've linked, and, as far as I can tell, it builds upon the existing webpack example and adds things that are needed specifically for building a VS Code extension.

I'm happy to accept changes that can be useful for multiple types of projects (e.g. https://github.com/NiGhTTraX/ts-monorepo/discussions/317), so let me know if you think that's the case here.

**Comment by syhxzzz** [comment]
Thank you for your time. After reviewing my attempt, I realized that I merely combined your apps/webpack and apps/cra into a single project. Therefore, there is no need to include this in the examples. I will close this issue.
